src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,47 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { FC } from 'react';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  tabId: string;
+  onTabSelect: (tab: Tab) => void;
+};
+
+export const Tabs: FC<Props> = ({ tabs, tabId, onTabSelect }) => {
+  const selectedTab = () => {
+    return tabs.find((tab: Tab) => tab.id === tabId) || tabs[0];
+  };
```

Comment:
Why do you need this like a func?
```suggestion
  const selectedTab = tabs.find((tab: Tab) => tab.id === tabId) || tabs[0];
```